ambient-mesh/migration/v1/gather-cluster-info.sh

```diff
@@ -0,0 +1,601 @@
+#!/bin/sh
```

Comment:
When I cloned this repo and tried to run this, I had to `chmod +x` this file - this is something we can commit to the repo.

Reply:
Ah thanks, I thought I ran this before 🤦🏼